src/test/resources/picocompletion-demo-help_completion.bash

```diff
@@ -4,7 +4,7 @@
 # =======================
 #
 # Bash completion support for the `picocompletion-demo-help` command,
-# generated by [picocli](http://picocli.info/) version %1$s.
+# generated by [picocli](http://picocli.info/) version 4.6.3-SNAPSHOT.
```

Comment:
Please leave the `%1$s` variable, it will be replaced by the latest version in the test.